src/index.js

```diff
@@ -55,7 +55,9 @@ export function app(state, actions, view, container) {
   function resolveNode(node) {
     return typeof node === "function"
       ? resolveNode(node(globalState, wiredActions))
-      : node || ''
+      : node != null 
```

Comment:
@jonlov Did you run `npm run format`? Not sure if prettier would do this.

Reply:
@jorgebucaran if you're interested in checking the format using the build/CI: https://github.com/hyperapp/fx/blob/7be65b2983a6b0c9e732840343be4b458d8e34e4/package.json#L27

Reply:
@jorgebucaran no I didn't. I wrote it like that. It looks nicer to have the empty string at the end. Is it wrong to do it like this?

Reply:
@jonlov  https://github.com/hyperapp/hyperapp/blob/master/CONTRIBUTING.md#style

> We use Prettier to format the code. If you don't have Prettier installed in your code editor or IDE, you can use npm run format before adding a commit.

Reply:
@jorgebucaran sorry, didn't know.